src/Microsoft.ApiDesignGuidelines.Analyzers/Microsoft.ApiDesignGuidelines.Analyzers.md

```diff
@@ -170,6 +240,34 @@ Severity: Warning
 
 Help: [https://msdn.microsoft.com/en-us/library/bb264484.aspx](https://msdn.microsoft.com/en-us/library/bb264484.aspx)
 
+### CA1065: Do not raise exceptions in unexpected locations ###
+
+A method that is not expected to throw exceptions throws an exception.
+
+Category: Design
+
+Severity: Warning
+
+Help: [https://msdn.microsoft.com/en-us/library/bb386039.aspx](https://msdn.microsoft.com/en-us/library/bb386039.aspx)
+
+### CA1066: Type {0} should implement IEquatable<T> because it overrides Equals ###
+
+Category: Design
+
+Severity: Warning
+
+### CA1067: Override Object.Equals(object) when implementing IEquatable<T> ###
+
```

Comment:
Ah! yes, this is more what I would have expected the title of CA1066 to look like.